### ISSUE TITLE
Unable to Run Without Debugging: Process exiting with code: -4058 signal: false

### ISSUE THREAD

**Issue by RomanKornev** [golang/vscode-go#999]
### What version of Go, VS Code & VS Code Go extension are you using?
- Run `go version` to get version of Go
	- go version go1.15.6 windows/amd64
- Run `gopls -v version` to get version of Gopls if you are using [the language server](https://github.com/golang/vscode-go/blob/master/docs/gopls.md).
	- golang.org/x/tools/gopls v0.5.4
- Run `code -v` or `code-insiders -v` to get version of VS Code or VS Code Insiders
	- 1.51.1 e5a624b788d92b8d34d1392e4c4d9789406efe8f x64
- Check your installed extensions to get the version of the VS Code Go extension
	- 0.19.0
- Run `go env` to get the go development environment details
	- set GO111MODULE=
set GOARCH=amd64
set GOBIN=
set GOCACHE=C:\Users\Roman\AppData\Local\go-build
set GOENV=C:\Users\Roman\AppData\Roaming\go\env
set GOEXE=.exe
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOINSECURE=
set GOMODCACHE=C:\Users\Roman\go\pkg\mod
set GONOPROXY=
set GONOSUMDB=
set GOOS=windows
set GOPATH=C:\Users\Roman\go
set GOPRIVATE=
set GOPROXY=https://proxy.golang.org,direct
set GOROOT=C:\Go
set GOSUMDB=sum.golang.org
set GOTMPDIR=
set GOTOOLDIR=C:\Go\pkg\tool\windows_amd64
set GCCGO=gccgo
set AR=ar
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=C:\Python\Go\go_playground\go.mod
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=C:\Users\Roman\AppData\Local\Temp\go-build223042854=/tmp/go-build -gno-record-gcc-switches

### Share the Go related settings you have added/edited
```jsonc
    // #region GO
    "go.useLanguageServer": true,
    // "go.languageServerFlags": [
    //     "-rpc.trace"
    // ],
    "go.languageServerExperimentalFeatures": {
        "diagnostics": true,
        "documentLink": true,
        "format": true
    },
    "[go]": {
        "editor.rulers": [100],
        "editor.insertSpaces": false,
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.organizeImports": true
        },
        // "editor.snippetSuggestions": "none"
    },
    "[go.mod]": {
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.organizeImports": true
        },
    },
    "gopls": {
        "completeUnimported": true,
        "deepCompletion": true,
        "usePlaceholders": true,
        // WARNING: significantly increases memory usage
        "staticcheck":true
    },
    // #endregion GO
```
### Describe the bug
Pressing Run Without Debugging on a simple program results in `Process exiting with code: -4058 signal: false`.
Running with debugging works fine. Running with `go run ./cmd/hello` is also fine.

### Steps to reproduce the behavior:
1. Install Go on Windows 10 x64
2. Install all the tools
3. Make a Hello World program
4. Run Without Debugging

### Screenshots or recordings
![image](https://user-images.githubusercontent.com/5807561/101204059-b24a7200-367c-11eb-9488-f05023314cbc.png)

**Comment by hyangah** [comment]
Looks like a duplicate of https://github.com/golang/vscode-go/issues/918. 
please try the go nightly (https://marketplace.visualstudio.com/items?itemName=golang.go-nightly) to see if this is fixed at head.

**Comment by hyangah** [comment]
The newly released version of extension (0.19.1) should fix the issue.
Thanks for being patient!

**Comment by RomanKornev** [comment]
Yes, it's fixed. Thanks!
